File: src/main/java/org/jesperancinha/rockstarts/rockstarsmanager/converters/ArtistsConverter.java
package org.jesperancinha.rockstarts.rockstarsmanager.converters;

import org.jesperancinha.rockstarts.rockstarsmanager.data.ArtistDto;
import org.jesperancinha.rockstarts.rockstarsmanager.model.Artist;

public class ArtistsConverter {
    public static ArtistDto toDto(Artist artist) {
        return ArtistDto.builder()
            .id(artist.getId())
            .name(artist.getName())
            .build();
    }

    public static Artist toData(ArtistDto artistDto) {
        return Artist.builder()
            .id(artistDto.getId())
            .name(artistDto.getName())
            .build();
    }
}

File: src/main/kotlin/org/jesperancinha/rockstarts/rockstarsmanager/converters/ArtistsConverter.kt
package org.jesperancinha.rockstarts.rockstarsmanager.converters

import org.jesperancinha.rockstarts.rockstarsmanager.data.ArtistDto
import org.jesperancinha.rockstarts.rockstarsmanager.model.Artist

object ArtistsConverter {
    fun toDto(artist: Artist?): ArtistDto {
        return ArtistDto.builder()
            .id(artist.getId())
            .name(artist.getName())
            .build()
    }

    @JvmStatic
    fun toData(artistDto: ArtistDto): Artist {
        return Artist.builder()
            .id(artistDto.id)
            .name(artistDto.name)
            .build()
    }
}